sdk/search/Azure.Search.Documents/src/Azure.Search.Documents.csproj

```diff
@@ -28,7 +31,8 @@
 
   <ItemGroup>
     <PackageReference Include="Azure.Core" />
-    <PackageReference Include="System.Text.Json" />
+    <PackageReference Include="System.ClientModel" />
```

Comment:
Let's remove this.   We get it transitively through `Azure.Core` and shouldn't need a direct reference.